wee_slack.py

```diff
@@ -917,6 +916,23 @@ def emoji_completion_cb(data, completion_item, current_buffer, completion):
         w.hook_completion_list_add(completion, ":" + e + ":", 0, w.WEECHAT_LIST_POS_SORT)
     return w.WEECHAT_RC_OK
 
+@utf8_decode
+def usergroups_completion_cb(data, completion_item, current_buffer, completion):
+    """
+    Adds all :-prefixed usergroups to completion list
```

Comment:
`:-prefixed` isn't right for usergroups.

Reply:
typo error I will fixed that.